playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -116,3 +116,12 @@
   template:
     src: clouds.yml.j2
     dest: /etc/openstack/clouds.yaml
+
+- name: Save Shipyard env variables to ~/.airshiprc
+  blockinfile:
+    path: "{{ ansible_user_dir }}/.airshiprc"
+    create: yes
+    mode: "0700"
```

Comment:
nit: As this is supposed to be sourced, not executed, `0600` should be enough here

Reply:
OK